content/commands/ft.sugget/index.md

```diff
@@ -30,6 +30,8 @@ categories:
 - oss
 - kubernetes
 - clients
+command_flags:
+- readonly
```

Comment:
This file actually displays with a second syntax section which is empty. I did notice a space after the slash in the closing highlight tag (not changed in this commit), but I don't know if this explains the glitch.

Reply:
It's like that on the live site as well. I know the fix. Thank you!